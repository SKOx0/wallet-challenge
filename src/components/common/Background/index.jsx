import styled from 'react-emotion';

export default styled('div')`
  background-color: ${(props) => props.color};
  height: 100vh;
`;
