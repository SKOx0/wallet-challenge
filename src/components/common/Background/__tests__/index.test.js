import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Background from '../index';

describe('<Background />', () => {
  it('deve ser igual ao snapshot', () => {
    const renderedComponent = shallow(<Background color="#000000" />);

    expect(toJson(renderedComponent)).toMatchSnapshot();
  });
});
