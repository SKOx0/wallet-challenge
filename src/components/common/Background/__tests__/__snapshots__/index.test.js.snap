// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Background /> deve ser igual ao snapshot 1`] = `
.emotion-0 {
  background-color: #000000;
  height: 100vh;
}

<div
  className="emotion-0"
  color="#000000"
/>
`;
